showpass/Showpass-E2E-tests
# TA-85 Verify 'Yearly membership group' checkout process

**Description:**
This user story will be verifying that a new yearly renewal membership group can be created and that a pre-defined limited number of membership tickets can get added to cart.
NOTE: we will not be purchasing the tickets, cause we need to be able to destroy the group at the end but rather will be checking the checkout process as far as we can.

Steps:

- Destroy all previously created groups
- Create a new membership group with yearly renewal 
- Predefine the max number of tickets that can be purchased
- Open just created group on the front end
- Add the max number of tickets to cart
- Proceed with the checkout
- DO NOT purchase the tickets
- Clean up the cart to ensure the group can get deleted

**Jira ticket:**
https://showpass.atlassian.net/browse/TA-85

**How to test:**
Run the following spec and ensure it passes 'cypress/e2e/admin/membership-levels.cy.js'

**Test execution results:**

<img width="807" alt="TA-85-all-tests-passed" src="https://github.com/showpass/Showpass-E2E-tests/assets/138615449/9269fd41-d005-4491-b804-7fe1195ecfac">
